.changeset/dirty-dragons-flash.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/select-input': minor
+'@commercetools-uikit/select-utils': minor
+'@commercetools-uikit/design-system': minor
+---
+
+Creating a new theme 'Quite' for the `SelectInput` component.
```

Comment:
Perhaps we want to avoid `theme` word here to prevent confusion.
Maybe we can explain about the new `appearance` property and its available values.

Reply:
Thanks Carlos! I agree it's better if we keep the word _appearance_ instead - https://github.com/commercetools/ui-kit/pull/2644/commits/bda3dab4b7ab0e6b50ffd2393d8afd9a61f8db08